### ISSUE TITLE
make foreign key constraints editable - especially for cascades

### ISSUE THREAD

**Issue by praveenweb** [hasura/graphql-engine#225]
Currently, we do not display the `ON UPDATE` and `ON DELETE` properties of a foreign key constraint on the console once it is created, nor do we have an option to update them.

The update can happen by sending the following sql query using `run_sql`: 
```sql
alter table <table_name> drop constraint <constraint_name>, add constraint <constraint_name> FOREIGN KEY (<this_column>) REFERENCES <remote_table>(<remote_column>) ON UPDATE <new_update_setting> ON DELETE <new_update_setting>;
```

**Comment by coco98** [comment]
@rikinsk @dsandip Moving this up the priority ladder because this is a super desired feature.

**Comment by dsandip** [comment]
@rikinsk Let's see how this fits in with #506

**Comment by rikinsk** [comment]
Fixed in #1854
